client/state/posts/selectors.js

```diff
@@ -115,7 +115,12 @@ export function getSitePostsLastPageForQuery( state, siteId, query ) {
 		return null;
 	}
 
-	return state.posts.queries[ siteId ].getNumberOfPages( query );
+	const pages = state.posts.queries[ siteId ].getNumberOfPages( query );
```

Comment:
Is there any chance `state.posts.queries[ siteId ]` could ever be null or undefined?